testing/database.rst

```diff
@@ -86,7 +86,7 @@ it's easy to pass a mock object within a test::
 
             // Now, mock the repository so it returns the mock of the employee
             $employeeRepository = $this
-                ->getMockBuilder(EntityManagerInterface::class)
+                ->getMockBuilder(EntityRepository::class)
```

Comment:
I wonder if we should not mock the `ObjectRepository` interface instead. We then would even not have to use the `disableOriginalConstructor()` call.